tests/fixtures.py

```diff
@@ -78,7 +80,7 @@ def graph(session):
 def users(session):
     users = {
         username: User.get_or_create(session, username=username)[0]
-        for username in ("gary", "zay", "zorkian", "oliver", "testuser")
+        for username in ("gary", "zay", "zorkian", "oliver", "testuser", "figurehead")
```

Comment:
I considered that for the name of the role, but decided that its connotations were a bit too nebulous.